### ISSUE TITLE
[BUG] Sitting Mode's height adjustment doesn't bring you to full height, and is seemingly random adjustment

### ISSUE THREAD

**Issue by JadianRadiator** [QuestCraftPlusPlus/QuestCraft#248]
**Describe the bug**
"Sitting Mode" add's a seemingly random length to your height.
Most of the time it hardly changes your height.
The remainder uncommonly being around an approximate 1.4 to 1.5 meters instead of the approximate 1.8 meters you're supposed to be.
There is no way to manually choose your new height.

**To Reproduce**
1. Know how tall the player is is supposed to be compared to a 2 block tall wall
2. Get in-game in v1.2
3. Be sitting down
4. turn on "sitting mode" by clicking the right trigger and right thumbstick in, multiple times if needed
5. compare the adjusted height with how tall a player is is supposed to be compared to a 2 block tall wall

**Expected behavior**
Arbitrarily set the vr height to the non-vr's normal ~1.8 meter height

**Headset Details (please complete the following information):**
 - Quest Build: 38.0.0.63.727.351486027
 - Device: Quest 2
 - QuestCraft Version: v1.2

**Additional context**
I generally have to either deal with being seemingly too short,
or have to use the Quest's "height adjust" accessibility feature.

**Comment by wellsilver** [comment]
just stand up smh

**Comment by Flopgop** [comment]
Stale.

**Comment by smartglasseshub** [comment]
As a VR enthusiast, I've explored ways to adjust height on the Quest 2 for a better experience, especially when playing seated. Here's a summary of what I found:

- **Adjust Height Setting:** Easily accessible in Quest 2 settings, this feature lets you virtually raise your height, improving interaction with VR games while seated.
- **Disabling Guardian & Tracking:** Involves toggling off the Guardian and tracking settings. This readjusts the headset's perception of your height.
- **Lowering the Floor Level:** By placing a controller at a lower level than your play area during setup, the floor level is tricked, making you appear taller in the game.
- **Controller Swing Method:** Swinging the controllers towards the floor during setup can mislead the headset into thinking the floor is lower, thus increasing your in-game height.
- **Choosing Suitable Games:** Some games have better mechanics for playing while seated or allow height adjustments in settings.

Hopefully this helped. Some of the answers are a bit more nuanced. Check out the full article on [my experience here](https://smartglasseshub.com/oculus-quest-2-height-adjustment/).
